### DISCUSSION TITLE
请教Windows下，在不开启系统代理的情况下，如何让qBittorrent-ClientBlocker使用代理？

### DISCUSSION THREAD

**Discussion by Mozart2045** [Simple-Tracker/qBittorrent-ClientBlocker#126] [General]
我这边访问不了https://cdn.jsdelivr.net/的all.txt和ipBlockList.txt，然后满屏的错误提示

开了系统代理，那自然就正常了，但是这台电脑不方便一直开着系统代理

config.json里，只找到了"proxy": "Auto"，怎么让qBittorrent-ClientBlocker访问http://127.0.0.1:1080这样代理端口？

（不会写代码，问题描述可能不专业，请见谅）
![qb](https://github.com/user-attachments/assets/a8ddbeaf-c018-43aa-af3d-f84c7cbfb1a4)

**Comment by Simple-Tracker** [comment]
使用批处理等方式创建一个环境, 手动设置环境变量, 它的优先级最高.

**Reply by Mozart2045** [reply]
感谢解答，能举个例子吗？（说实话，我没太懂...）

**Reply by Simple-Tracker** [reply]
start.bat
```
set HTTP_PROXY="http://127.0.0.1:7890"
set HTTPS_PROXY="http://127.0.0.1:7890"
qBittorrent-ClientBlocker.exe
```

**Reply by Mozart2045** [reply]
非常感谢,晚上回去试试

**Reply by Mozart2045** [reply]
再次感谢,靠着chatgpt写了一个bat,把启动qBittorrent和启动qBittorrent-ClientBlocker,并且给qBittorrent-ClientBlocker设置本地代理端口以及隐藏窗口到托盘一次性解决了
